apps/books/models.py

```diff
@@ -12,6 +14,16 @@ class Meta:
         verbose_name = 'Category'
         verbose_name_plural = 'Categories'
 
+    def get_absolute_url(self):
+        return reverse("books", kwargs={
+            'slug': self.slug
+        })
```

Comment:
Please don't mix single and double quotes.. Please use single quotes

Reply:
Secondly these 3 lines can be done on single line

Reply:
Okay